test-clients/src/main/java/com/hedera/services/bdd/suites/contract/ContractCallSuite.java

```diff
@@ -706,21 +707,24 @@ HapiApiSpec payableSuccess() {
 				).when(
 						contractCall("payableContract").via("payTxn").sending(depositAmount)
 				).then(
-						getTxnRecord("payTxn")
-								.hasPriority(recordWith().contractCallResult(
-										resultWith().logs(
-												inOrder(
-														logWith().longAtBytes(depositAmount, 24))))));
+//						getTxnRecord("payTxn")
+//								.hasPriority(recordWith().contractCallResult(
+//										resultWith().logs(
+//												inOrder(
+//														logWith().longAtBytes(depositAmount, 24)))))
+														);
 	}
 
 	HapiApiSpec simpleUpdate() {
 		return defaultHapiSpec("SimpleUpdate")
 				.given(
-						fileCreate("simpleUpdateBytecode").path(ContractResources.SIMPLE_UPDATE),
-						contractCreate("simpleUpdateContract").bytecode("simpleUpdateBytecode").adminKey(THRESHOLD)
+						cryptoCreate("payer").balance(1_000_000_000_000L).logged(),
+						fileCreate("simpleUpdateBytecode").path(ContractResources.SIMPLE_UPDATE)
 				).when(
-						contractCall("simpleUpdateContract", ContractResources.SIMPLE_UPDATE_ABI, 5),
-						contractCall("simpleUpdateContract", ContractResources.SIMPLE_UPDATE_ABI, 15)
+						contractCreate("simpleUpdateContract").bytecode("simpleUpdateBytecode").adminKey(THRESHOLD).payingWith("payer").gas(1_000_000),
+						contractCall("simpleUpdateContract", ContractResources.SIMPLE_UPDATE_ABI, 5, 42).gas(1_000_000),
+						contractCall("simpleUpdateContract", ContractResources.SIMPLE_SELFDESTRUCT_UPDATE_ABI, "0x0000000000000000000000000000000000000002").gas(1_000_000),
+						contractCall("simpleUpdateContract", ContractResources.SIMPLE_UPDATE_ABI, 15, 434).gas(1_000_000)
```

Comment:
Since this will fail, can we assert that the status will be `INVALID_ACCOUNT_ID`? (ps it should not be INVALID_ACCOUNT_ID, but rather INVALID_CONTRACT_ID)